lib/r2d2/payment_token.rb

```diff
@@ -27,7 +30,8 @@ def decrypt(private_key_pem)
       # verify the tag is a valid value
       self.class.verify_mac(digest, hkdf_keys[:mac_key], encrypted_message, tag)
 
-      self.class.decrypt_message(encrypted_message, hkdf_keys[:symmetric_encryption_key])
+      message = self.class.decrypt_message(encrypted_message, hkdf_keys[:symmetric_encryption_key])
+      JSON.parse(message)
```

Comment:
The current contract of r2d2 is that it tries to assume as little as possible about how the caller wants to parse the resulting message. This has the added benefit of letting the caller use their own JSON parser, or defer to a later point in processing to do so. I've found that people are quite opinionated about their chosen JSON lib :) I'd like to retain that contract here, and just return the message string itself, letting the caller decide what they do with it from there.